src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -223,6 +225,19 @@ export default class SourceFileLinter {
 			])) {
 				return;
 			}
+
+			const nonStaticRender = (node as ts.ClassDeclaration).members.find((member: ts.ClassElement) => {
+				return (ts.isPropertyDeclaration(member) || ts.isMethodDeclaration(member)) &&
+					(ts.isIdentifier(member.name) || ts.isStringLiteral(member.name)) &&
```

Comment:
I wonder whether the check for equality works when the import was a relative import (`import MyRenderer from "./MyRenderer.js"`)

Reply:
Is this test case what you meant: https://github.com/SAP/ui5-linter/pull/392/commits/976992b348a09d5ca4de9a85f0acd42827523475 ?

If, yes, then yes, it works even with relative imports

Reply:
Not sure if this proves that the check works as expected.

To my understanding, the issue in `7ControlRenderer` is already found because of `7Control`. So what is the indication that the import of `7ControlRenderer` by `Test5` also leads to an analysis of `7ControlRenderer`? 

Shouldn't this rather be a new renderer (which is not reached otherwise) and if it appears in the snapshot, then it's working?

Well, but I might have missed something.

Reply:
This check is done, but not reported as it is a duplicate entry and we clean duplicates.
I'll create a separate renderer file. Thanks!

Reply:
https://github.com/SAP/ui5-linter/pull/392/commits/102b7b1e5904a17298d3d23f3fbc05f7e9454b74